internal/storage/s3/s3.go

```diff
@@ -17,11 +18,16 @@ import (
 	"github.com/offen/docker-volume-backup/internal/storage"
 )
 
+type s3StorageInternal struct {
+	defaultPartSize int64
+}
+
 type s3Storage struct {
 	*storage.StorageBackend
 	client       *minio.Client
 	bucket       string
 	storageClass string
+	internal     s3StorageInternal
```

Comment:
What's the idea of introducing an `internal` type instead of just putting `defaultPartSize` on the top level `s3Storage` struct?